{{cookiecutter.project_slug}}/.travis.yml

```diff
@@ -40,6 +41,16 @@ script:
 
 # Report coverage
 after_success:
+  - if [[ "${PYVER}" == "3.6" ]];
+    then
+        source activate root;
+        conda remove -y --force -n testenv --all;
+        conda create -y -n testenv python="3.5";
+        conda remove -y --force -n testenv openssl;
+        conda env update -n testenv --file environment_ci.yml;
+        conda list --full-name -n testenv openssl || conda install -y -n testenv openssl;
+        source activate testenv;
+    fi
```

Comment:
Downgrade to Python 3.5 when submitting coverage to coveralls. See commit ( https://github.com/jakirkham/kenjutsu/pull/51/commits/3ca2bf3c89c2448a2c92a8f5ab02b49b3dcf356a ) for details.